lib/sdf-server/src/service/component.rs

```diff
@@ -118,16 +118,31 @@ impl IntoResponse for ComponentError {
             | ComponentError::NotFound(_) => (StatusCode::NOT_FOUND, self.to_string()),
             ComponentError::PropertyEditor(err) => match err {
                 PropertyEditorError::ComponentNotFound
+                | PropertyEditorError::PropertyEditorValueNotFoundByPropId(_)
                 | PropertyEditorError::SchemaVariantNotFound(_) => {
                     (StatusCode::NOT_FOUND, err.to_string())
                 }
-
+                PropertyEditorError::AttributePrototype(_)
```

Comment:
Are we sure this is a LOOP_DETECTED?

Reply:
It's bad request, you've misread the rust I think. Loop detected is the next match arm